app/assets/stylesheets/object_types.css.scss

```diff
@@ -168,11 +168,22 @@
 
 .object-types {
   .selected {
-    background: #80d8ff;
+    background: #B5D7EA;
   }
   .flash {
     padding: 10px;
     width: 100%;
-    background: lightyellow;
+    background:#d2f8d2;
   }  
 }
+
+
+// Updating 'Upload Data' button
```

Comment:
Comments should be to clarify what element the style is for. 
Please don't describe the changes you've made here, instead use any comments to clarify.  In this case, maybe say `Style for 'Upload Data' button`